src/lib/eccentric.scss

```diff
@@ -427,89 +428,55 @@ input[type=date], input[type=datetime] {
   border: none;
   border-radius: 4px;
 }
-// input[type=image],input[type=month]{
-//   border: none;
-//   border-radius: 4px;
-// }
-
-
-/// Implementing the .et-header Component - For Desktop Views
-
-// .et-header, .et-.et-header {
-//   padding: 1em;
-//   background-color: var(--et-black);
-//   overflow: hidden;
-//   border: inherit;
-//   border-radius: inherit;
-// }
-
-// .et-.et-header-option {
-//   color: white;
-//   font-family: var(--et-description);
-//   list-style: none;
-//   display: inline;
-//   margin-right: 2em;
-//   cursor: pointer;
-// }
-
-// .et-.et-header-option:hover {
-//   color: gray;
-// }
-
-// @media screen and (max-width: 1000px) {
-//   .et-header, .et-.et-header {
-//     text-align: center;
-//   }
-// }
 
 .et-header {
   overflow: hidden;
-  background-color: #333;
+  background-color: #292929;
```

Comment:
Don't directly use the color code, the root-CSS-variables are defined in the same module.

Instead of using `background-color: #292929;`
we should be using `background-color: var(--et-black);`